source/localizable/getting-started/quick-start.md

```diff
@@ -211,13 +211,6 @@ curly braces (`{{component}}`). We're going to tell our component:
    provide this route's `model` as the list of people.
 
 ```app/templates/scientists.hbs{-1,-2,-3,-4,-5,-6,-7,+8}
```

Comment:
Ahh now I see what the intention is ... it tries to show that lines from 1 to 7 should be removed and line 8 is added. But that's not showing properly here https://guides.emberjs.com/v2.3.0/getting-started/quick-start/